redisson/src/main/java/org/redisson/client/handler/CommandEncoder.java

```diff
@@ -59,6 +63,21 @@ public class CommandEncoder extends MessageToByteEncoder<CommandData<?, ?>> {
     private static final char BYTES_PREFIX = '$';
     private static final byte[] CRLF = "\r\n".getBytes();
 
+    private static final Integer STRING_CACHE_SIZE = 100_000;
```

Comment:
This may lead to much memory allocation. Please change it to 10_000.

Reply:
Good point, changed